engineering/continuous-delivery.md

```diff
@@ -0,0 +1,93 @@
+Web application deployment workflow.
+====
+
+<!-- TOC -->
+
+- [Intro](#intro)
+- [Quick summary:](#quick-summary)
+    - [branches](#branches)
+    - [Continuous delivery](#continuous-delivery)
+- [General workflow](#general-workflow)
+- [Deploying into Production](#deploying-into-production)
+    - [special cases](#special-cases)
+        - [Release branch](#release-branch)
+        - [Exceptions](#exceptions)
+- [Deploying into Staging](#deploying-into-staging)
+
+<!-- /TOC -->
+
+# Intro
+
+This documentation assumes it is already known the [compay deployment workflow](git-flow.md), and extends it to cover the continuous delivery as it should work for all source{d} web applications:
+
+repo | Prod | Staging
+--- | --- | ---
+[src-d/landing](https://github.com/src-d/landing) | [sourced.tech](https://sourced.tech) | [landing-staging.srcd.run](https://landing-staging.srcd.run)
+[src-d/blog](https://github.com/src-d/blog) | [blog.sourced.tech](http://blog.sourced.tech) | [blog-staging.srcd.run](https://blog-staging.srcd.run)
+[src-d/talks](https://github.com/src-d/talks) | [talks.sourced.tech](https://talks.sourced.tech) | [talks-staging.srcd.run](https://talks-staging.srcd.run)
+
+This document does not pretend to be exhaustive but a guide, and there can be special cases that will be defined in the README.md and CONTRIBUTING.md of each web application repository.
+
+This document is based on the agreements of the [2017-10-23 CD workflow](https://github.com/src-d/minutes/blob/master/infrastructure/2017-10-23-landing-cd-workflow.md) and [2017-10-26 Enforcing the CD workflow](https://github.com/src-d/minutes/blob/master/infrastructure/2017-10-26-enforcing-cd-workflow.md) meetings.
+
+# Quick summary:
+
+Here are the technical details of when CI or CD runs, but it is needed you know and understand the guides and conventions explained below this section.
+
+## branches
+- `master` branch must be "always" production ready.
```

Comment:
What do quotations marks mean here? Also, `production ready` can be interpreted in different ways. I reckon that what we agreed is that `master` branch's tip will match the deployed code running in production environment.